src/public/js/ui/app.tsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Footer } from './components/footer';
 import { Header } from './components/header';
 import { LoginForm } from './components/login-form';
+import InspectLet from '../vendor/inspectlet';
 
 export const App: React.FC = () => {
+  useEffect(() => {
+    if(window.inspectlet_key){
+      InspectLet();
+    }
+  }, [])
```

Comment:
There might be some formatting issues here

Reply:
I think indentation is correct here, solved the naming convention issue, can you please tell me what mistake needs to be fixed here??